packages/amazon-sumerian-hosts-babylon/webpack.dev.js

```diff
@@ -11,7 +11,7 @@ module.exports = merge(common, {
     modules: [path.resolve(__dirname, '../../node_modules'), 'node_modules'],
   },
   devServer: {
-    open: 'Google Chrome',
+    open: ['/examples'],
```

Comment:
nit: just want to point out i think we do still have some of the integration test in babylon although i'm ok with ignoring them for now

Reply:
Ah. I'll add a dedicated npm script for launching the integration tests.

Reply:
This issue is addressed in the update I just posted.